src/main/java/solvd/laba/ermakovich/hf/kafka/XmlUtils.java

```diff
@@ -0,0 +1,27 @@
+package solvd.laba.ermakovich.hf.kafka;
+
+
+import com.jcabi.xml.XML;
+import com.jcabi.xml.XMLDocument;
+import lombok.SneakyThrows;
+import lombok.experimental.UtilityClass;
+import org.springframework.util.ResourceUtils;
+
+
+/**
+ * @author Ermakovich Kseniya
+ */
+
+@UtilityClass
+public class XmlUtils {
```

Comment:
Try to read [this](https://www.yegor256.com/2017/02/07/private-method-is-new-class.html)